components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -205,4 +212,16 @@ private void removeSessionDataFromCache(String sessionDataKey) {
         OIDCSessionDataCacheKey cacheKey = new OIDCSessionDataCacheKey(sessionDataKey);
         OIDCSessionDataCache.getInstance().clearCacheEntry(cacheKey);
     }
+
+    /**
+     * Returns the OpenIDConnect User Consent.
+     *
+     * @return
+     */
+    private static boolean getOpenIDConnectSkipeUserConsent() {
+
+        boolean errorPageUrl = OAuthServerConfiguration.getInstance().getOpenIDConnectSkipeUserConsentConfig();
```

Comment:
Why not just user the return statement here rather than assigning to a new variable